pihole-google.txt

```diff
@@ -7399,4 +7405,293 @@ www.tf-enterprise-devel.gcp.fitbit.com
 www.vps.store-au.fitbit.com
 www.wiki.fitbit.com
 www.windchill.fitbit.com
-wwwinternal.fitbit.com
\ No newline at end of file
+wwwinternal.fitbit.com
+android-safebrowsing.googleapis.com
+developers.google.com
+safebrowsing.google.com
+safebrowsing.googleapis.com
+sb-ssl.google.com
+sb.l.google.com
+acceleratedmobilepageurl.googleapis.com
+android.bugly.qq.com
+app-measurement.com
+autofill.googleapis.com
+bugly.qq.com
```

Comment:
qq.com domains are not linked to Google. These should be removed

Reply:
Done